<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pro_QuaryBal</class>
 <widget class="QDialog" name="Pro_QuaryBal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="line0_card">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>108</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelHint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卡号信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询余额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>53</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请刷卡&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>268</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_times">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>53</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>[60]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2_pwd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>268</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入密码：</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line0_card</tabstop>
  <tabstop>line2_pwd</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
